e2e-tests/test/main.test.ts

```diff
@@ -11,6 +12,19 @@ async function deployGreeter(deployer: Deployer): Promise<Contract> {
   return await deployer.deploy(artifact, ['Hi']);
 }
 
+async function fundAccount(
+  wallet: ethers.Wallet,
+  address: string,
+  amount: string,
+) {
+  await (
+    await wallet.sendTransaction({
+      to: address,
+      value: ethers.utils.parseEther(amount),
+    })
+  ).wait();
+}
+
```

Comment:
I'd suggest moving this to a `utils` file so other tests in the future may make use of.

Reply:
If we want to invest more into hardhat e2e tests, then we can do something similar to the structure proposed in this PR: https://github.com/matter-labs/era-test-node/pull/92/files#diff-9e76579c32b2c0011636cc3ce790c039a7c34887e5ca3adbfcd7c2e06cf4d250

I'm just not convinced we'll be adding more e2e tests going forward. Thoughts?